src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -58,13 +58,19 @@
           :data-test-id "status-icon" :data-test-value "unknown"}
    (spinner)])
 
-(defn icon-for-cluster-status [status]
-  (case status
-    ("Deleted" "Error") (moncommon/render-failure-icon)
-    ("Creating" "Updating" "Deleting" "Stopping" "Starting") spinner-icon
-    "Running" (moncommon/render-success-icon)
-    "Stopped" nil  ;todo: add icon for Stopped status
-    (moncommon/render-unknown-icon)))
+(def pause-icon
+  [:span {:style {:display "inline-flex" :alignItems "center" :justifyContent "center" :verticalAlign "middle"
+                  :width table-style/table-icon-size :height table-style/table-icon-size
+                  :borderRadius 3 :margin "-4px 4px 0 0"}
+          :data-test-id "status-icon" :data-test-value "unknown"}
+   (icons/render-icon {} :pause)])
```

Comment:
`render-icon` returns a span, so I think you should be able to collapse these two into one.